receiver/hostmetricsreceiver/README.md

```diff
@@ -191,3 +191,18 @@ Currently, the hostmetrics receiver does not set any Resource attributes on the
 export OTEL_RESOURCE_ATTRIBUTES="service.name=<the name of your service>,service.namespace=<the namespace of your service>,service.instance.id=<uuid of the instance>"
 ```
 
+## Processes scraper deprecation
+
+The `processes` scraper has been deprecated in favor of the `process` scraper. The `processes` scraper will be removed in a future release. To enable the same functionality, remove the `processes` scraper and enable the `system.processes.*` metrics in the `process` scraper:
+
+```yaml
+receivers:
+  hostmetrics:
+    scrapers:
+      process:
+        metrics:
+          system.processes.created:
```

Comment:
The changes on the metrics have already been merge in SemConv with https://github.com/open-telemetry/semantic-conventions/pull/484.

I wonder if we are planning to first deprecate the scraper and then in a follow-up PR adjust the metrics as well. Is that the plan?

Reply:
The original plan discussed was to make this change first, keeping it compatible with the old metric names. This was so that it would be easier to manage during the semconv transition period.